x-pack/plugin/core/src/main/config/log4j2.properties

```diff
@@ -51,7 +52,7 @@ appender.audit_rolling.layout.pattern = {\
 # "user.run_by.realm" the realm name of the impersonating subject ("user.run_by.name")
 # "user.run_as.realm" if this "event.action" is of a run_as type, this is the realm name the impersonated user is looked up from
 # "user.roles" the roles array of the user; these are the roles that are granting privileges
-# "origin.type" it is "rest" if the event is originating (is in relation to) a REST request; possible other values are "transport" and "ip_filter"
+# "event.type" it is "rest" if the event is originating (is in relation to) a REST request; possible other values are "transport", "ip_filter" and "security_config_change"
```

Comment:
The `origin.type` and `event.type` definitions have been reversed in this configuration.